.circleci/config.yml

```diff
@@ -140,13 +67,18 @@ workflows:
           requires:
             - build
 
-      - e2eTestAppBasic:
+      - architect/integration-test:
+          name: basic-integration-test
+          test-dir: "integration/test/app/basic"
           requires:
             - build
```

Comment:
This is so nice man! Good job!

Reply:
Thanks Pawel! Happy this finally shipped and your initial PR and reviews helped a ton.